src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -207,26 +207,913 @@
 
                 <controls:SettingsGroup IsEnabled="True" x:Uid="ShortcutGuide_Appearance_Behavior">
                     <controls:Setting x:Uid="ColorModeHeader" Icon="&#xE790;">
-                                <controls:Setting.Description>
-                                    <HyperlinkButton Click="OpenColorsSettings_Click"
+                        <controls:Setting.Description>
+                            <HyperlinkButton Click="OpenColorsSettings_Click"
                                                      x:Uid="Windows_Color_Settings"/>
-                                </controls:Setting.Description>
+                        </controls:Setting.Description>
+                        <controls:Setting.ActionContent>
+                            <ComboBox SelectedIndex="{x:Bind Mode=TwoWay, Path=ViewModel.ThemeIndex}" MinWidth="{StaticResource SettingActionControlMinWidth}">
+                                <ComboBoxItem x:Uid="Radio_Theme_Dark"/>
+                                <ComboBoxItem x:Uid="Radio_Theme_Light"/>
+                                <ComboBoxItem x:Uid="Radio_Theme_Default"/>
+                            </ComboBox>
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+                    <controls:Setting x:Uid="GeneralPage_RunAtStartUp">
+                        <controls:Setting.ActionContent>
+                            <ToggleSwitch IsOn="{Binding Mode=TwoWay, Path=Startup}" x:Uid="ToggleSwitch"/>
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+                </controls:SettingsGroup>
+
+                <!-- ShortcutGuide_SettingsBackupAndRestore -->
+                <!--vTEST Start Option A-->
```

Comment:
Let's clean up to have only the one we use.

Reply:
Removed all extra/testing versions.